### ISSUE TITLE
TODO: Cloning Console GUI and basic Cloning Operating System B: 200

### ISSUE THREAD

**Issue by DooblyNoobly** [unitystation/unitystation#1729]
# Criteria

This job is to begin work on the Cloning Console GUI and to build the foundation of the Cloning Operating System. When a player clicks on the Cloning Computer the Cloning Console GUI should appear as a pop up NetTab (see how it is handled via GUI_Paper).

This Job is for creating the Graphical Interface to use the Cloning Machine and for wiring up all the buttons so they trigger actions on the server for these functions:

- Scan a body to capture DNA sequence
- Save DNA sequence in Cloning Computer
- Lock / Unlock DNA scanner
- Start Cloning

As we are waiting on the Drag and Drop feature to be implemented we just want these functions to call methods on the server, the actual cloning logic will come in a later TODO. Please place Logger.Logs in the functions for the time being to prove you have hooked up the buttons correctly.

Please make sure the system has all the basic functionality as seen in the reference screen shots below:

Img References from TGStation:

First screen:
![image](https://user-images.githubusercontent.com/20813925/56002588-5ba15f00-5d06-11e9-8a37-bc8ec2ec1a00.png)

Records List Screen:
![image](https://user-images.githubusercontent.com/20813925/56002623-7b388780-5d06-11e9-8d01-67b4daede307.png)

Record View Screen:
![image](https://user-images.githubusercontent.com/20813925/56002671-9d320a00-5d06-11e9-9ddd-9b06512a2545.png)

Make sure your Record system works by creating some test entries (you can do this by using ContextMenus - See UnityDocs) and also ensure they delete properly in the record view. 

Also test to make sure the records remain for anyone who accesses the Cloning Console.

It is also important to test this feature with a dedicated headless server and 2 clients (Tick the TestServer flag on the GameData gameobject in the first scene of your build, then untick it for your clients. The dedicated server will automatically start a server when you run it on localhost).

Contributor will receive a bounty of 200usd on a successful merge of a solution that satisfies the criteria above.

**Comment by Aranclanos** [comment]
Working on this but its on hold till https://github.com/unitystation/unitystation/issues/1837 gets solved

**Comment by DooblyNoobly** [comment]
Bumping up the bounty on this one as it is a top priority. Would be good to get it in before end of the month if possible.

**Comment by fomalsd** [comment]
Done by @Aranclanos in #1906
